.github/workflows/check-build-installers.yml

```
@@ -0,0 +1,27 @@
+name: "[Check] Build installers without certs"
```

Also I would name this a bit differently
```suggestion
name: "[Build] unsigned installers and binaries"
```

---

ok